peakina/readers/csv.py

```diff
@@ -0,0 +1,57 @@
+"""
+Module to add csv support
+"""
+from typing import Any, Dict, Optional
+
+import pandas as pd
+
+# The chunksize value for previews
+PREVIEW_CHUNK_SIZE = 1024
+
+
+def read_csv(
+    filepath: str,
+    *,
+    sep: str = ",",
+    keep_default_na: Any = None,
+    encoding: Optional[str] = None,
+    preview: Dict[str, int] = {},
```

Comment:
```suggestion
    preview: Optional[PreviewArgs]
```
with
```py
class PreviewArgs(TypedDict, total=False):
    nrows: int
    offset: int
```
would be better probably

Reply:
👍🏾  Updated !